### ISSUE TITLE
Switching to more concise mime types

### ISSUE THREAD

**Issue by boris-glumpler** [alexdebril/feed-io#393]
Hello @alexdebril,

would you be open to a PR that returns a more appropriate mime type? Currently, a feed response will have either an `application/json` or `application/xhtml+xml` type, it seems.

The more concise ones would be:

```
application/feed+json
application/rss+xml
application/atom+xml
```

I don't really know what the support of these mimes looks like in feed readers, tho. The RSS and Atom mimes should probably be good to go, but Json Feed only switched about 1.5 years ago.

**Comment by alexdebril** [comment]
Hi @boris-glumpler 
Thanks, you can submit it whenever you want !

I think your proposal is cleaner than what feed-io actually provides, so it's a good idea. I'm not expecting readers to break because of this, it's a rather expectable behaviour so it should be OK 👌

Thanks for your support ❤️

**Comment by alexdebril** [comment]
shipped in version 5.1.0
